ArduCopter/system.cpp

```diff
@@ -452,13 +452,19 @@ void Copter::allocate_motors(void)
         ac_var_info = AC_AttitudeControl_Multi_6DoF::var_info;
 #endif // AP_SCRIPTING_ENABLED
     } else {
-        attitude_control = new AC_AttitudeControl_Multi(*ahrs_view, aparm, *motors, scheduler.get_loop_period_s());
-        ac_var_info = AC_AttitudeControl_Multi::var_info;
+        #if CUSTOMCONTROL_ENABLED == ENABLED
+            attitude_control = new AC_CustomControl(*ahrs_view, aparm, *motors, scheduler.get_loop_period_s());
+            ac_var_info = AC_CustomControl::var_info;
+        #else
+            attitude_control = new AC_AttitudeControl_Multi(*ahrs_view, aparm, *motors, scheduler.get_loop_period_s());
+            ac_var_info = AC_AttitudeControl_Multi::var_info;
```

Comment:
need to use              
```
 ac_var_info = AC_AttitudeControl_Multi::var_info;
```
for both cases, so the load_object_from_eeprom gets the whole subtree